src/Serilog.Sinks.MSSqlServer/Serilog.Sinks.MSSqlServer.csproj

```diff
@@ -30,13 +30,16 @@
 
   <ItemGroup>
     <!-- Whenever these are updated, also update versions in packages.config for net452 -->
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.0.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Serilog" Version="2.5.0" />
-    <PackageReference Include="Serilog.Sinks.PeriodicBatching" Version="2.3.0" />
+    <PackageReference Include="Serilog" Version="2.10.0" />
+    <PackageReference Include="Serilog.Sinks.PeriodicBatching" Version="2.3.1" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="4.0.1" />
```

Comment:
4.1 version of [Microsoft.Data.SqlClient  is now availible](https://github.com/dotnet/SqlClient/blob/main/release-notes/4.1/4.1.0.md) , it does not contain many new features and should not affect this library.

@ckadluba  Let me know if you want any further upgrades (via code suggestion, comments or by pushing changes)
I am not planning to make any more changes to this PR unless there are any request to do so.